core/Test/Tasty/Core.hs

```diff
@@ -335,6 +335,10 @@ testGroup = TestGroup
 -- order, all dependencies will be run (overriding filter).
 -- For parallel execution, see 'testGroup'.
 --
+-- Note that this is will only work when used with a `TestManager`. If you use
```

Comment:
"with the default `TestManager`"
(everything is a TestManager, `launchTestTree` is the default one)